build.gradle

```diff
@@ -75,7 +75,7 @@ dependencies {
 	javadocClasspath "net.fabricmc:fabric-loader:${project.fabric_loader_version}"
 	javadocClasspath "org.jetbrains:annotations:${project.jetbrains_annotations_version}"
 	javadocClasspath "com.google.code.findbugs:jsr305:3.0.2" // for some other jsr annotations
-	decompileClasspath "org.benf:cfr:0.150"
+	decompileClasspath "org.benf:cfr:0.151"
```

Comment:
This should use Fabric fork of CFR, and put the version in gradle.properties

Reply:
Fixed in https://github.com/FabricMC/yarn/pull/2343/commits/39985abb91ac989c3d6876208d2bc5dd34cc78df